lib/validate_encrypt_utils.pm

```diff
@@ -80,19 +141,29 @@ sub verify_cryptsetup_properties {
     }
 }
 
+=head2 verify_restoring_luks_backups 
+
+  verify_restoring_luks_backups($mapped_dev, $mapped_dev_path, $backup_file_info, $backup_path);
+￼ 
+￼ C<mapped_dev_path>: path of encrypted device
```

Comment:
Could you please remove these strange `￼` characters?

Reply:
I could not distinguish what was the strange character from the comment and it is not visible in my pr for some reason. But i believe that they were some `emacs` things. I removed all i found but let me know if you see any again